branchinfo.txt

```diff
@@ -6,6 +6,6 @@ MINOR_VERSION=1
 PATCH_VERSION=0
 STABILIZE_PACKAGE_VERSION=false
 LOCK_HOST_VERSION=false
-RELEASE_SUFFIX=preview1
+RELEASE_SUFFIX=preview2
```

Comment:
Can we just delete it now?  No one uses it, and it has been deleted in release/2.0.0